src/logcollector/config.c

```diff
@@ -195,6 +195,9 @@ void _getLocalfilesListJSON(logreader *list, cJSON *array, int gl) {
             cJSON_AddNumberToObject(multiline, "timeout", list[i].multiline->timeout);
             cJSON_AddItemToObject(file, "multiline_regex", multiline);
         }
+        if (list[i].regex_ignore) cJSON_AddStringToObject(file, "regex_ignore", w_expression_get_regex_pattern(list[i].regex_ignore));
```

Comment:
```suggestion
        if (list[i].regex_ignore) cJSON_AddStringToObject(file, "ignore", w_expression_get_regex_pattern(list[i].regex_ignore));
```

The name included in this JSON must match the name of the setting.